RATIONALE.md

```diff
@@ -1105,3 +1105,21 @@ In lieu of formal documentation, we infer this pattern works from other sources
 See https://github.com/golang/go/blob/go1.20/src/sync/waitgroup.go#L64
 See https://github.com/golang/go/issues/5045#issuecomment-252730563
 See https://www.youtube.com/watch?v=VmrEG-3bWyM
+
+## Advanced Wazero Features
+
+### Why enable "hugepages" globally instead of per wazero.Runtime instance?
```

Comment:
I don't really like this here because it is experimental. I feel like this is tail wagging the dog into us normalizing ENV tricks as a normal case. Can you please move this note to experimental because the feature is still experimental?